valid-anagram/whewchews.ts

```diff
@@ -0,0 +1,21 @@
+function isAnagram(s: string, t: string): boolean {
+  // TC: O(1)
+  if (s.length !== t.length) return false;
+
+  // SC: O(N)
+  const count: { [key: string]: number } = {};
+
+  // TC: O(N)
+  for (let i = 0; i <= s.length - 1; i++) {
+    const sChar = s[i];
+    const tChar = t[i];
+    count[sChar] = (count[sChar] || 0) + 1;
+    count[tChar] = (count[tChar] || 0) - 1;
```

Comment:
nit: 이 문제의 경우 무방하지만, 일반적으로는 `||` 대신에 `??`를 사용하는 것이 더 안전하지 않을까 생각이 들었습니다.

```suggestion
    count[sChar] = (count[sChar] ?? 0) + 1;
    count[tChar] = (count[tChar] ?? 0) - 1;
```

Reply:
감사합니다! 말씀해주신대로 ?? 을 사용하는게 좋을 것 같습니다.
이런 코드에서는 원하는 값이 없는 경우에, 기본값을 사용할 목적이니 falsy한 값을 모두 처리하는 ||(논리OR연산자)가 아닌 null이나 undefined 값만 처리하는 ??(널병합연산자) 를 사용하는게 더 목적에 맞고 안전하게 처리해줄 수 있겠네요!